Tests/GDSAnalyticsTests/Screens/ErrorScreenViewTests.swift

```diff
@@ -11,11 +11,14 @@ final class ErrorScreenViewTests: XCTestCase {
     }
     
     func testEmptyParametersAreRemoved() {
-        let view = ErrorScreenView(screen: MockScreen.error,
+        let uuid = UUID().uuidString.lowercased()
+
+        let view = ErrorScreenView(id: uuid,
+                                   screen: MockScreen.error,
                                    titleKey: "Something went wrong")
         
         XCTAssertEqual(view.parameters, [
-            "title": "something went wrong"
+            "screen_id": uuid
```

Comment:
please can we ensure that the Analytics team are aware that this change may affect the ID Check data in the next few weeks?

Reply:
I assume they wont see the title until updated to the new schema?

Reply:
Sure, but we should warn them it's coming in a few weeks when we make that change and check if there's any impact (i.e. when all the "screen class" values suddenly change).